src/env/tock.rs

```diff
@@ -157,15 +157,15 @@ pub fn switch_off_leds() {
 }
 
 const KEEPALIVE_DELAY_MS: isize = 100;
-pub const KEEPALIVE_DELAY: Duration<isize> = Duration::from_ms(KEEPALIVE_DELAY_MS);
+pub const KEEPALIVE_DELAY_TOCK: Duration<isize> = Duration::from_ms(KEEPALIVE_DELAY_MS);
```

Comment:
Couldn't we reduce the diff by not renaming here?

Reply:
The problem is in the main.rs, there are 2 places using KEEPALIVE_DELAY. With this PR, we will need one of them using duration of embedded_time and the other with tock duration. That is why I made the change here. I believe in the future PR, this will not be a problem.